scan/lib/scan/options.rb

```diff
@@ -69,6 +69,13 @@ def self.available_options
                                      type: Boolean,
                                      optional: true),
 
+        # reset simulator
+        FastlaneCore::ConfigItem.new(key: :reset_simulator,
+                                     env_name: 'SCAN_RESET_SIMULATOR',
+                                     description: "Enabling this option will automatically erase the simulator before running the application",
+                                     default_value: false,
+                                     is_string: false),
```

Comment:
Can we make this change? 🙃 `is_string` will soon be gone
```suggestion
                                     type: Boolean),
```

Reply:
Good to know. Yes please go ahead and change it. Thanks.